src/components/AssetCard/AssetCard.css

```
@@ -0,0 +1,368 @@
+  margin: 0px;
+  flex: none;
+}
+
+.AssetCard.ui.cards > .ui.card.AssetCard.link .meta,
+.AssetCard.ui.card.link .meta {
+  margin-bottom: 4px;
+}
+
+.AssetCard .tags {
+  margin-top: 10px;
+}
+
+.AssetCard .sub-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  color: var(--text);
+}
+
+.AssetCard .card-meta {
+  margin-right: 10px;
+}
+.AssetCard .NotForSale {
+  font-weight: bold;
+  margin-top: 12px;
+  margin-bottom: 4px;
+}
+
+.AssetCard .PriceInMana .ui.header.large {
+  font-size: 30px;
+  font-weight: 600;
+  margin: 0px;
+  display: flex;
+  flex-direction: row;
+}
+
+.AssetCard .ens-subdomain {
+  overflow: hidden;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.AssetCard .dcl.atlas {
+  overflow: hidden;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+}
+
+.ui.card.AssetCard .AssetImage,
+.ui.card.AssetCard .AssetImage .rarity-background {
+  border-radius: 10px 10px 0px 0px !important;
+}
+
+.AssetCard .AssetImage .ens-subdomain {
+  background-size: 557px;
+  background-position-y: -133px;
+}
+
+.AssetCard .AssetImage .ens-subdomain .monospace {
+  margin-top: 0px;
+}
+
+.AssetCard .AssetImage {
+  height: 223px;
+  transition: height 0.3s !important;
+}
+
+.AssetCard .AssetImage.catalog {
+  height: 209px;
+  transition: height 0.3s !important;
+}
+
+.AssetCard .AssetImage.remove-margin {
+  margin-bottom: unset;
+}
+
+.AssetCard .extraInformation {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+@media (min-width: 1199px) {
+  .AssetCard .extraInformation {
+    display: flex;
+    visibility: hidden;
+    height: 0px;
+    opacity: 0;
+    transition: height 0.1s, opacity 0.6s;
+  }
+
+  .AssetCard:hover .AssetImage.catalog {
+    height: 161px;
+    transition: none;
+    transition: height 0.3s !important;
+  }
+
+  .ui.card.AssetCard:hover .AssetImage.catalog,
+  .ui.cards > .card.AssetCard:hover .AssetImage.catalog {
+    box-shadow: 0px 4px 34px 0px rgba(255, 255, 255, 0.37) !important;
+    border-radius: 10px !important;
+  }
+
+  .AssetCard:hover .extraInformation {
+    visibility: visible;
+    margin-top: 5px;
+    height: auto;
+    opacity: 1;
+    align-items: center;
+  }
+
+  .AssetCard.ui.cards > .ui.card.AssetCard.link:hover .meta,
+  .AssetCard.a.ui.card.link:hover .meta {
+    color: var(--secondary-text);
+  }
+
+  .AssetCard:hover .CatalogItemInformation {
+    margin-top: 2px;
+  }
+
+  .AssetCard.ui.cards a.card.AssetCard.catalog:hover,
+  .AssetCard.ui.link.card.AssetCard.catalog:hover,
+  .AssetCard.ui.link.cards .card.AssetCard.catalog:hover,
+  .AssetCard.a.ui.card.AssetCard.catalog:hover,
+  .AssetCard.ui.cards > .ui.card.AssetCard.catalog.link:hover,
+  .AssetCard.ui.card.AssetCard.catalog.link:hover {
+    border: none !important;
+  }
+
+  .AssetCard.ui.card.AssetCard:hover > .content.catalog.expandable,
+  .AssetCard.ui.cards > .card.AssetCard:hover > .content.catalog.expandable {
```

Everything related to `.ui.cards` is meant to style the cards when being used in a container that has the `.ui.cards` class, like the `Card.Group` component. If it has the `AssetCard` class, it won't work, as the card is the one that has the `AssetCard` class and not the `Card.Group` component.
```suggestion
  .ui.cards > .card.AssetCard:hover > .content.catalog.expandable {
```
In my opinion, the concept of styling a parent component from a child one is dangerous as it can cause issues when placing the component in other containers. We can leave it here for now, but we need to change it eventually.